proto/swapi.proto

```diff
@@ -77,6 +78,8 @@ service Starwars {
        get: "/swapi/v1/people"
     };
   }
+
+  rpc ListStarshipActions(google.protobuf.Empty) returns (stream StarshipAction) {}
```

Comment:
I have a whole rant I'll write out in the evolution chapter about not using `Empty` since it prevents you from changing the request object later without breaking existing clients. Can we make an empty message called `StarShipActionRequest` so it matches the others?

Reply:
Yeah I'll do that.